GameData/RealismOverhaul/RO_SuggestedMods/AviationCockpits/Cockpits.cfg

```diff
@@ -247,3 +434,88 @@
 		%kerbalEyeOffset = 0.0, 0.0, 0.0
 	}
 }
+
+//Sukhoi Su30
+@PART[Mk1?Su30?Cockpit]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@title = Su-30 Cockpit
+	@manufacturer = Sukhoi
+	@description = Twin seat fighter cockpit.
+	%rescaleFactor = 1.2 // 1.5m end diameter
+
```

Comment:
Nice catch! Didn't notice that bc the node was in the right place, just the wrong way around